Rmarkdown/Multiple_Ascending_Dose_PD_continuous.Rmd

```diff
@@ -1,314 +1,311 @@
+    )
+#create a factor for the treatment variable for plotting
+pkpd_data = pkpd_data %>%
+  arrange(DOSE) %>%
+  mutate(TRTACT_low2high = factor(TRTACT, levels = unique(TRTACT)),
+         TRTACT_high2low = factor(TRTACT, levels = rev(unique(TRTACT))))
+
+#create pd dataset
+pd_data <- pkpd_data %>%
+  filter(CMT==3)
+
+
+#units and labels
+  time_units_dataset = "hours"
+  time_units_plot    = "days"
+  trtact_label       = "Dose"
+  dose_label         = "Dose (mg)"
+  conc_label         = "Concentration (ng/ml)"  
+  pd_label           = "Continuous PD Marker (IU/L)"  
+  concnorm_label     = "Normalized Concentration (ng/ml)/mg"
+  conc_units         = "ng/ml"
+  AUC_units          = "AUCtau (h.ng/ml)"
+  
+#directories for saving individual graphs
+dirs = list(
+  parent_dir= tempdir(),
+  rscript_dir  = "./",
+  rscript_name = "Example.R",
+  results_dir  = "./",
+  filename_prefix   = "",
+  filename     = "Example.png")
+```
+
+## Provide an overview of the data
+
+Summarize the data in a way that is easy to visualize the general trend of PD over time and between doses. Using summary statistics can be helpful, e.g. Mean +/- SE, or median, 5th & 95th percentiles. Consider either coloring by dose or faceting by dose. Depending on the amount of data one graph may be better than the other.
+
+When looking at summaries of continuous PD over time, there are several things to observe. Note the number of doses and number of time points or sampling schedule. Observe the overall shape of the average profiles. How does the profile change over time? Does the effect appear to increase and decrease quickly on a short time scale, or does is occur over a longer time scale? Is there clear separation between the profiles for different doses? Does the effect appear to increase with increasing dose? Do you detect a saturation of the effect? 
+
+### PD marker over time, colored by Dose, mean (95% CI) percentiles by nominal time
+
+```{r, cache = TRUE, fig.width = 8, fig.height = 4, warning = FALSE, message= FALSE} 
+
+#PD data
+gg <- ggplot(data = pd_data, 
+               aes(x=NOMTIME/24,y=LIDV, color = TRTACT_high2low, fill = TRTACT_high2low)) 
+gg <- gg + xgx_stat_ci(conf_level = .95)
+
+gg <- gg + xgx_scale_x_time_units(units_dataset = time_units_dataset, 
+                                  units_plot    = time_units_plot)
+gg <- gg + guides(color=guide_legend(""),fill=guide_legend(""))
+gg <- gg + scale_x_continuous(breaks=seq(-1,9,1))
+gg <- gg + scale_y_continuous()
+gg <- gg + labs(y=pd_label)
+gg <- gg + xgx_annotate_status(status)
+
+#if saving copy of figure, replace xgx_annotate lines with xgx_save() shown below:
+#xgx_save(width, height, dirs, "filename_main", status)
+print(gg)
+
+```
+
+### PD marker over time, faceted by Dose, mean (95% CI) percentiles by nominal time
+
+```{r, cache = TRUE, fig.width = 10, fig.height = 3, warning = FALSE, message= FALSE}
+ 
+gg <- ggplot(data = pd_data %>% subset(DOSE>0), 

```

Comment:
remove this text: %>% subset(DOSE>0)
don't need to filter out placebo group

Reply:
ok, re-added back